update.go

```diff
@@ -302,3 +304,12 @@ other options are ignored.
 		return container.Set(config)
 	},
 }
+
+// Since the OCI spec is designed for cgroup v1, in some cases
+// there is need to convert from the cgroup v1 configuration to cgroup v2
+// the formula for cpuShares is y = (1 + ((x - 2) * 9999) / 262142)
+// convert from [2-262144] to [1-10000]
+// 262144 comes from Linux kernel definition "#define MAX_SHARES (1UL << 18)"
+func convertCPUSharesToCgroupV2Value(cpuShares uint64) uint64 {
+	return (1 + ((cpuShares-2)*9999)/262142)
+}
```

Comment:
All these convertion functions should be put in a utils file so it'll be easier to remove them when we have these items in runtime-spec someday.

I've reclaimed my LGTM in the other PR.

Reply:
Make sense!